cli/checkpoint/lua/play.lua

```diff
@@ -0,0 +1,143 @@
+-- This is a script that play the contents of .snap/.xlog files to another Tarantool instance.
+-- The files and uri passes throught "TT_CLI_PLAY_FILES_AND_URI".
+-- The --show-system flag passes throught "TT_CLI_PLAY_SHOW_SYS".
+-- The --space flags passes throught "TT_CLI_PLAY_SPACES".
+-- The --from flag passes throught "TT_CLI_PLAY_FROM".
+-- The --to flag passes throught "TT_CLI_PLAY_TO".
+-- The --replica flags passes throught "TT_CLI_PLAY_REPLICAS".
+
+local log = require('log')
+local xlog = require('xlog')
+local json = require('json')
+local netbox = require('net.box')
+local argparse = require('internal.argparse').parse
+
+local function find_in_list(id, list)
+    if type(list) == 'number' then
+        return id == list
+    end
+    for _, val in ipairs(list) do
+        if val == id then
+            return true
+        end
+    end
+    return false
+end
+
+local function filter_xlog(gen, param, state, opts, cb)
+    local from, to, spaces = opts.from, opts.to, opts.space
+    local show_system, replicas = opts['show-system'], opts.replica
```

Comment:
How about making `--show-system` default behaviour and add `--hide-system` instead? I found `tarantoolctl cat` default behaviour confusing.

@ochaton, what do you think?

Reply:
@GRISHNOV I suggest that you discuss this issue with @Totktonada  in a voice vote and move forward (write down your decision here).

Reply:
Normally, user wants only his spaces (IMHO). I think we should add flag --show-system